websites/A/AsphaltEvents/dist/metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "Senka Wolf",
+      "id": "170854083840704512"
+    },
+    "contributors": {
+      "name": "DubStepMad",
+      "id": "118810952530395136"
+    },
```

Comment:
contributor needs to be in an array `"contributors": [{object}],`

Reply:
_Should be your last needed change_

Reply:
Resolved in updated pull request.